Blobbers get deregister due to the health check events failure

Blobbers might be marked as offline after running for a while due to the error shown below, that's why system test case `TestBlobberConfigUpdate` failed, because it could not find any blobbers.

```sh
2023-01-10T10:37:11.578Z        ERROR   event/process.go:192    error processing event  {"round": 2679, "tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blob
bers\" violates not-null constraint (SQLSTATE 23502)"}
2023-01-10T10:37:12.619Z        ERROR   event/process.go:278    addStat typeStats error {"round": 2684, "block": "5b201e08b81615ac1de309af40395f6216721d47d2466c78862405301308ae3a", "block size": 3, "event t
ype": "TagUpdateBlobberAllocatedHealth", "event tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blobbers\" violates not-null constraint (SQLSTATE 23502)"}
2023-01-10T10:37:12.619Z        ERROR   event/process.go:319    event could not be processed    {"round": 2684, "block": "5b201e08b81615ac1de309af40395f6216721d47d2466c78862405301308ae3a", "block size": 3,
"event type": "TagUpdateBlobberAllocatedHealth", "event tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blobbers\" violates not-null constraint (SQLSTATE 235
02)"}
2023-01-10T10:37:12.619Z        ERROR   event/process.go:192    error processing event  {"round": 2684, "tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blob
bers\" violates not-null constraint (SQLSTATE 23502)"}
2023-01-10T10:37:14.704Z        ERROR   event/process.go:278    addStat typeStats error {"round": 2694, "block": "7c4cf77b98d2fd7deee3af795e943ac90f4b1462220fbcc44e08a863913df8d7", "block size": 3, "event t
ype": "TagUpdateBlobberAllocatedHealth", "event tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blobbers\" violates not-null constraint (SQLSTATE 23502)"}
2023-01-10T10:37:14.704Z        ERROR   event/process.go:319    event could not be processed    {"round": 2694, "block": "7c4cf77b98d2fd7deee3af795e943ac90f4b1462220fbcc44e08a863913df8d7", "block size": 3,
"event type": "TagUpdateBlobberAllocatedHealth", "event tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blobbers\" violates not-null constraint (SQLSTATE 235
02)"}
2023-01-10T10:37:14.704Z        ERROR   event/process.go:192    error processing event  {"round": 2694, "tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blob
bers\" violates not-null constraint (SQLSTATE 23502)"}
2023-01-10T10:37:16.169Z        ERROR   event/process.go:278    addStat typeStats error {"round": 2701, "block": "c2700dfbdd38e146f43cca6db369708746dce3a7061510519e1c33c6614b080c", "block size": 3, "event t
ype": "TagUpdateBlobberAllocatedHealth", "event tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blobbers\" violates not-null constraint (SQLSTATE 23502)"}
2023-01-10T10:37:16.169Z        ERROR   event/process.go:319    event could not be processed    {"round": 2701, "block": "c2700dfbdd38e146f43cca6db369708746dce3a7061510519e1c33c6614b080c", "block size": 3,
"event type": "TagUpdateBlobberAllocatedHealth", "event tag": "TagBlobberHealthCheck", "error": "ERROR: null value in column \"bucket_id\" of relation \"blobbers\" violates not-null constraint (SQLSTATE 23$
```

--- a/code/go/0chain.net/smartcontract/dbs/event/provider.go
+++ b/code/go/0chain.net/smartcontract/dbs/event/provider.go
@@ -73,6 +73,7 @@ func (edb *EventDb) updateProvidersHealthCheck(updates []dbs.DbHealthCheck, tabl
 	updateExpr := map[string]interface{}{
 		"last_health_check": gorm.Expr("excluded.last_health_check"),
 		"downtime": gorm.Expr(fmt.Sprintf("%v.downtime + excluded.downtime", tableName)),
+		"bucket_id": gorm.Expr(fmt.Sprintf("%v.bucket_id", tableName))
 	}
 
 	return edb.Store.Get().Clauses(clause.OnConflict{

--- a/code/go/0chain.net/smartcontract/dbs/event/provider.go
+++ b/code/go/0chain.net/smartcontract/dbs/event/provider.go
@@ -28,7 +28,7 @@ type Provider struct {
 	ID             string `gorm:"primaryKey"`
 	CreatedAt      time.Time
 	UpdatedAt      time.Time
-	BucketId       int64           `gorm:"not null"`
+	BucketId       int64           `gorm:"not null,default:0"`
 	DelegateWallet string          `json:"delegate_wallet"`
 	MinStake       currency.Coin   `json:"min_stake"`
 	MaxStake       currency.Coin   `json:"max_stake"`
@@ -73,7 +73,6 @@ func (edb *EventDb) updateProvidersHealthCheck(updates []dbs.DbHealthCheck, tabl
 	updateExpr := map[string]interface{}{
 		"last_health_check": gorm.Expr("excluded.last_health_check"),
 		"downtime": gorm.Expr(fmt.Sprintf("%v.downtime + excluded.downtime", tableName)),
-		"bucket_id": gorm.Expr(fmt.Sprintf("%v.bucket_id", tableName))
 	}
 
 	return edb.Store.Get().Clauses(clause.OnConflict{

